externs/shaka/player.js

```diff
@@ -943,7 +944,11 @@ shaka.extern.xml.Node;
  * @property {Object.<string, string>} keySystemsByURI
  *   A map of scheme URI to key system name. Defaults to default key systems
  *   mapping handled by Shaka.
- * @property {function(!shaka.extern.xml.Node)} manifestPreprocessor
+ * @property {function(!Element)} manifestPreprocessor
```

Comment:
What we typically do with deprecated configs is to remove them from the config structure, and convert old to new with quoted strings and shims in lib/player.js.  See `'forceTransmuxTS'` in `lib/player.js` for an example.

I'm okay with doing it differently, but doing it this way makes these old configs show up in the docs.  If we're going to do this, let's add a deprecation notice here in the docs.  Something like `<b>DEPRECATED</b>: Use manifestPreprocessorTXml instead` would work for me.